README.md

```diff
@@ -7,3 +7,4 @@ Also, one of them had a very, very BIG secret; she could turn into a cupcake if
 Anyways, back to the story…
 They were both going shopping one day then one of them got lost.
 They also forgot where they parked their shopping cart with all of their merchandise in it.
+They looked around the entire store for an hour looking for the cart
```

Comment:
Would it be possible to add a period to the end of the sentence?

```suggestion
They looked around the entire store for an hour looking for the cart.
```

Otherwise, this looks great!